### PR TITLE
TypeScript

### PR DESCRIPTION
**Pull request by castilh0s**
## Proposed Changes

Oh my god, what's happening? The architecture is changing again? This is impossible!

But that's the true, I'm inserting TypeScript into project because types is love, types is life.

### Related PR

This PR already implement #35

### CODE REVIEW

**Comment by izn** [comment]
Too long, didn't read.

1. We need to add a new line at the end of all files. _Maybe your editor is misconfigured?_
2. We should update `.travis.yml` config step from `config.js` to `config.ts`.

File: src/views/Confirmation/Confirmation.view.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  }
+
+  private async handleConfirm() {
+    const { navigation } = this.props
+    const { user, product } = this.state
+
+    this.setState({ isActivityIndicatorAnimating: true })
+
+    try {
+
```
Comment (markx3) [inline src/views/Confirmation/Confirmation.view.tsx]:
```suggestion
```

File: src/views/Confirmation/Confirmation.view.tsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+      await ConfirmationService.createTransaction({
+        created_at: TimeHelper.currentDate(),
+        user: user.name,
+        product: product.name,
+        price: product.price,
+      })
+
+      Alert.alert(`Compra efetuada.\nObrigado, ${user.name}! 🥳`)
+    } catch (error) {
+
```
Comment (markx3) [inline src/views/Confirmation/Confirmation.view.tsx]:
```suggestion
```

File: src/views/Confirmation/Confirmation.view.tsx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+      Alert.alert('Ocorreu um erro ao confirmar a compra. 🤕')
+    } finally {
+
```
Comment (markx3) [inline src/views/Confirmation/Confirmation.view.tsx]:
```suggestion
```

**Pull request merged**